src/NuGet.Licenses/NuGet.Licenses.csproj

```diff
@@ -95,26 +95,47 @@
     <Folder Include="Models\" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Autofac.Mvc5">
+      <Version>4.0.2</Version>
+    </PackageReference>
+    <PackageReference Include="Microsoft.ApplicationInsights">
+      <Version>2.8.1</Version>
+    </PackageReference>
     <PackageReference Include="Microsoft.ApplicationInsights.Web">
-      <Version>2.4.1</Version>
+      <Version>2.8.1</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.AspNet.Mvc">
-      <Version>5.2.4</Version>
+      <Version>5.2.6</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.AspNet.Razor">
-      <Version>3.2.4</Version>
+      <Version>3.2.6</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.AspNet.Web.Optimization">
       <Version>1.1.3</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.AspNet.WebPages">
-      <Version>3.2.4</Version>
+      <Version>3.2.6</Version>
+    </PackageReference>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions">
+      <Version>2.1.1</Version>
```

Comment:
>2.1.1 [](start = 15, length = 5)

We use v1 Microsoft.Extensions everywhere else. Do you envision we'll need to use ServerCommon/NuGetGallery packages in this project? If so, we may want to downgrade these versions.